src/status_actions.c

```diff
@@ -678,7 +676,7 @@ bool8 sub_805BA44(Entity * pokemon, Entity * target, Move *move, s32 param_4)
     return TRUE;
 }
 
-bool8 sub_805BA50(Entity * pokemon, Entity * target, Move *move, s32 param_4)
+bool8 TakeawayMoveAction(Entity * pokemon, Entity * target, Move *move, s32 param_4)
```

Comment:
Takeaway is used by the Mug Orb.

Reply:
Yes and no. This also gets called for Thief and Covet.

sub_805A2A0 (Thief and Covet) will call this function. Might name sub_805A2A0 `ThiefMoveAction`.

Reply:
Ah okay, in that case naming it after Thief would be more recognizable than naming it after Takeaway.